wrappers/python/simtk/openmm/app/pdbxfile.py

```diff
@@ -201,3 +203,125 @@ def getPositions(self, asNumpy=False, frame=0):
                 self._numpyPositions[frame] = Quantity(numpy.array(self._positions[frame].value_in_unit(nanometers)), nanometers)
             return self._numpyPositions[frame]
         return self._positions[frame]
+
+    @staticmethod
+    def writeFile(topology, positions, file=sys.stdout, keepIds=False,
+                  entry=None):
+        """Write a PDB file containing a single model.
+
+        Parameters:
+         - topology (Topology) The Topology defining the model to write
+         - positions (list) The list of atomic positions to write
+         - file (file=stdout) A file to write to
+         - keepIds (bool=False) If True, keep the residue and chain IDs specified in the Topology rather than generating
+           new ones.  Warning: It is up to the caller to make sure these are valid IDs that satisfy the requirements of
+           the PDB format.  Otherwise, the output file will be invalid.
+         - entry (str=None) The entry ID to assign to the CIF file
+        """
+        PDBxFile.writeHeader(topology, file, entry)
+        PDBxFile.writeModel(topology, positions, file, keepIds=keepIds)
+
+    @staticmethod
+    def writeHeader(topology, file=sys.stdout, entry=None):
+        """Write out the header for a PDB file.
+
+        Parameters:
+         - topology (Topology) The Topology defining the molecular system being written
+         - file (file=stdout) A file to write the file to
+         - entry (str=None) The entry ID to assign to the CIF file
+        """
+        if entry is not None:
+            print('data_%s' % entry, file=file)
+        else:
+            print('data_cell', file=file)
+        print("# Created with OpenMM %s, %s" % (Platform.getOpenMMVersion(), str(date.today())), file=file)
+        print('#', file=file)
+        vectors = topology.getPeriodicBoxVectors()
+        if vectors is not None:
+            (a, b, c) = vectors.value_in_unit(angstroms)
```

Comment:
I didn't know about it.  I adapted `PDBFile` for this, and `computeLengthsAndAngles` wasn't used there, either.

I can change both if you want.